client/css/style.css

```diff
@@ -915,6 +915,13 @@ button, .user {
 	padding-top: 2px;
 	line-height: 1.8;
 }
+#settings #change-password .error {
+	color: #e74c3c;
+	margin-bottom: 1em;
+}
+#settings #change-password .error.success {
+	color: #2ecc40;
```

Comment:
- Why the `margin-bottom` for the error but not the success message? It doesn't seem necessary for none I think, but I can be wrong.
- `.error.success`? :-)